### DISCUSSION TITLE
Please could you explain how to set container and boundariesElement properties?

### DISCUSSION THREAD

**Discussion by Redrica** [Akryum/v-tooltip#378] [Q&A] [answered]
_container - Selector: Container where the tooltip will be appended (e.g. 'body').
boundariesElement - DOM element for the tooltip boundaries._

what kind of selector I need?
I tried to use a classname there but it didnt work.
I have a set of cards with tooltips in them and I want to place tooltips' element inside a card instead of body.
and the same question about boundariesElement.

Thanks in advance!

**Comment by luispittagros** [comment, marked as answer]
I think you must do something like 

```js
const boundariesElement =  document.querySelector('#boundariesElementl');
```
